man/repart.d.xml

```diff
@@ -789,6 +789,32 @@
 
         <xi:include href="version-info.xml" xpointer="v256"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>Compression=</varname></term>
+
+        <listitem><para>Specify the compression algorithm to use for the filesystem configured with
+        <varname>Format=</varname>. Takes a single argument specifying the compression algorithm. </para>
+
+        <para>Note that this setting is only taken into account when the filesystem configured with
+        <varname>Format=</varname> supports compression (btrfs, squashfs, erofs).</para>
+
```

Comment:
I think if we add this, the help text here should actively useful. i.e. say that the supported compression algorithms and their identifying names vary between file systems, and than add a brief table or so listing known algorithms for each fs at this time. No need to be complete though. And maybe add a sentence emphesizing that this is table is not supposed to be comprehensive, and further algorithms might be supported, and that people should consult the relevant docs for the actual list of algorithms.

i.e.:

file system | compression algorithms         | documentation
------------|--------------------------------|---------------
squashfs    | gzip, lzo, lz4, xz, zstd, lzma | mksquashfs(1)